starfish/util/config.py

```diff
@@ -0,0 +1,60 @@
+import collections
+import json
+import os
+from typing import Any, Dict, Sequence, Union
+
+
+class Config(collections.UserDict):
```

Comment:
I'm on the fence whether being a UserDict is good or not...

Reply:
Other options you're considering?

Reply:
I dunno, I would be okay with just invoking `Config.lookup` for everything.

Reply:
Other changes pushed. Time to decide on this.

Reply:
Pushed a commit to look at no `UserDict`